include/picolibrary/microchip/megaavr0/peripheral/bod.h

```diff
@@ -0,0 +1,436 @@
+      public:
+        /**
+         * \brief Field sizes.
+         */
+        struct Size {
+            static constexpr auto LVL       = std::uint_fast8_t{ 3 }; ///< LVL.
+            static constexpr auto RESERVED3 = std::uint_fast8_t{ 5 }; ///< RESERVED3.
+        };
+
+        /**
+         * \brief Field bit positions.
+         */
+        struct Bit {
+            static constexpr auto LVL = std::uint_fast8_t{}; ///< LVL.
+            static constexpr auto RESERVED3 = std::uint_fast8_t{ LVL + Size::LVL }; ///< RESERVED3.
+        };
+
+        /**
+         * \brief Field bit masks.
+         */
+        struct Mask {
+            static constexpr auto LVL = mask<std::uint8_t>( Size::LVL, Bit::LVL ); ///< LVL.
+            static constexpr auto RESERVED3 = mask<std::uint8_t>( Size::RESERVED3, Bit::RESERVED3 ); ///< RESERVED3.
+        };
+
+        /**
+         * \brief LVL.
+         */
+        enum LVL : std::uint8_t {
+            LVL_BODLEVEL0 = 0x0 << Bit::LVL, ///< 1.8 V.
+            LVL_BODLEVEL2 = 0x2 << Bit::LVL, ///< 2.6 V.
+            LVL_BODLEVEL7 = 0x7 << Bit::LVL, ///< 4.2 V.
+        };
+
+        CTRLB() = delete;
+
+        CTRLB( CTRLB && ) = delete;
+
+        CTRLB( CTRLB const & ) = delete;
+
+        ~CTRLB() = delete;
+
+        auto operator=( CTRLB && ) = delete;
+
+        auto operator=( CTRLB const & ) = delete;
+
+        using Register<std::uint8_t>::operator=;
+    };
+
+    /**
+     * \brief VLM Control A (VLMCTRLA) register.
+     *
+     * This register has the following fields:
+     * - VLM Level (VLMLVL)
+     */
+    class VLMCTRLA : public Register<std::uint8_t> {
+      public:
+        /**
+         * \brief Field sizes.
+         */
+        struct Size {
+            static constexpr auto VLMLVL    = std::uint_fast8_t{ 2 }; ///< VLMLVL.
+            static constexpr auto RESERVED2 = std::uint_fast8_t{ 6 }; ///< RESERVED2.
+        };
+
+        /**
+         * \brief Field bit positions.
+         */
+        struct Bit {
+            static constexpr auto VLMLVL = std::uint_fast8_t{}; ///< VLMLVL.
+            static constexpr auto RESERVED2 = std::uint_fast8_t{ VLMLVL + Size::VLMLVL }; ///< RESERVED2.
+        };
+
+        /**
+         * \brief Field bit masks.
+         */
+        struct Mask {
+            static constexpr auto VLMLVL = mask<std::uint8_t>( Size::VLMLVL, Bit::VLMLVL ); ///< VLMLVL.
+            static constexpr auto RESERVED2 = mask<std::uint8_t>( Size::RESERVED2, Bit::RESERVED2 ); ///< RESERVED2.
+        };
+
+        /**
+         * \brief VLMLVL.
+         */
+        enum VLMLVL : std::uint8_t {
+            VLMLVL_5ABOVE = 0x0 << Bit::VLMLVL, ///< VLM threshold 5% above BOD threshold.
+            VLMLVL_15ABOVE = 0x1 << Bit::VLMLVL, ///< VLM threshold 15% above BOD threshold.
+            VLMLVL_25ABOVE = 0x2 << Bit::VLMLVL, ///< VLM threshold 25% above BOD threshold.
+        };
+
+        VLMCTRLA() = delete;
```

Comment:
Enum is not defined by the datasheet.
```suggestion
        };

        VLMCTRLA() = delete;
```